### PR TITLE
Add more resources about colors

### PR DESCRIPTION
**Pull request by okonet**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -195,6 +199,10 @@ A curated list of links on component-driven development and design systems
 - [styled-tools](https://github.com/diegohaz/styled-tools) — useful interpolated functions for styled-components and other CSS in JS libraries
 - [styled-system](https://github.com/jxnblk/styled-system) — design system utilities for styled-components and other CSS in JS libraries
 
+### Colors
+
+- https://github.com/lyft/coloralgorithm
```
Comment (sapegin) [inline Readme.md]: Could you add a description for this one?

**Review by sapegin — CHANGES_REQUESTED** [review]
Thanks!

**Comment by sapegin** [comment]
And a link as a link, and not a URL ;-/

**Comment by okonet** [comment]
I think GitHub is going to convert it to link automatically

**Comment by okonet** [comment]
2019-01-31 at 14 50

**Comment by sapegin** [comment]
It's a clickable URL and it's ugly. You've just spent more time making this screenshot, than fixing would take ;-/

**Comment by okonet** [comment]
I've changed it

**Pull request merged**